client/src/ui-components/ProductCardComponent/ProductCard.js

```diff
@@ -1,58 +1,183 @@
 import React from "react";
-import { Container, Card, CardImg, Row, Col, Button } from "react-bootstrap";
-
+import {
+  Container,
+  Card,
+  CardImg,
+  Row,
+  Col,
+  Button,
+  ProgressBar,
+  Form,
+  FormControl,
+} from "react-bootstrap";
+import {
+  iconPackaging,
+  iconCartPlus,
+  iconDelete,
+  iconCartEmptySmall,
+} from "../icons";
 import "./ProductCard.css";
 import "bootstrap/dist/css/bootstrap.min.css";
 import ImageService from "../../services/ImageService/ImageService";
 
 function ProductCard(props) {
   const product = props.product;
+  let left_availability = 14; //mock left_availability
+  let dropdown_items = [];
+  for (let i = 1; i < 11; i++) {
+    if (i <= left_availability) {
+      dropdown_items.push(i);
+    }
+  }
 
   return (
-    <Container>
-      <Card>
-        <CardImg src={ImageService.returnImageByCategory(product.category)} />
+    <Container className="px-2">
+      <Card className="card">
+        <CardImg
+          src={ImageService.returnImageByCategory(product.category)}
+          className="card-img"
+        />
         <Card.Body className="body">
-          <Card.Title className="card-title">{product.name}</Card.Title>
-          <Row>
-            <Col>
-              <Card.Text className="text">
-                {product.description}
-                <br />
-                Packaging: {product.availability.packaging}
-                <br />
-                Price: {product.availability.price} €
+          <div className="min-height mx-0 px-0">
+            <Row className="d-flex justify-content-between mb-1">
+              <Col xs="8" sm="7">
+                <Card.Title className="card-title">{product.name}</Card.Title>
+              </Col>
+              <Col className="product-price" sm="5" xs="4">
+                {product.availability.price} €
+              </Col>
+            </Row>
+
+            <Row className="mb-3 pt-1">
+              <Card.Text className="card-text">
+                {iconPackaging}
+                <span className="packaging-text">
+                  Packaging: {product.availability.packaging}
+                </span>
               </Card.Text>
-            </Col>
+            </Row>
+            <Row className="description">
+              <Card.Text className="card-text">{product.description}</Card.Text>
+            </Row>
+          </div>
+
+          <Row className="mt-4 progressbar-row">
+            {props.shoppingCart.get(product.id) ? (
+              <>
+                <ProgressBar
+                  now={left_availability + props.shoppingCart.get(product.id)}
```

Comment:
```suggestion
                  now={left_availability - props.shoppingCart.get(product.id)}
```
Otherwise the left availability increases by adding element to the cart!